etc/config/nix.defaults.properties

```diff
@@ -0,0 +1,15 @@
+# Default settings for nix
+compilers=&nix
+compilerType=nix
+
+group.nix.compilers=nixdefault
+compiler.nixdefault.exe=/home/node/.nix-profile/bin/nix
```

Comment:
This is a little bit of a problem.

In Single-user nix installation, it gets installed in ~/.nix-profile/bin/nix. 
In nixos, it gets symlinked to :  /run/current-system/sw/bin/nix  -> which is a symlink to the nix store nix
In multi-user nix installation, it gets installed in /nix/var/nix/profiles/default/bin/nix

I put there the path of the codespace where I was, but it could be a little tricky to select a safe default.